steps/delius/contact/create-contact.ts

```diff
@@ -1,22 +1,26 @@
 import { expect, type Page } from '@playwright/test'
 import { findContactsByCRN } from './find-contacts.js'
 import { fillDate, fillTime, selectOptionAndWait } from '../utils/inputs.js'
-import { Contact, data, Team } from '../../../test-data/test-data.js'
+import {Contact, data, Staff, Team} from '../../../test-data/test-data.js'
 import { doUntil } from '../utils/refresh.js'
 
-export const createContact = async (page: Page, crn: string, options: Contact) => {
+export const createContact = async (page: Page, crn: string, options: { date: Date; allocation: { staff: Staff; team: Team; providerName: string }; category: string; type: string; relatesTo: string }) => {
```

Comment:
I don't think you need this change - it looks like it's just redefining the Contact class?